homeassistant/components/lcn/quality_scale.yaml

```diff
@@ -0,0 +1,75 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
```

Comment:
```suggestion
  docs-configuration-parameters:
    status: exempt
    comment: Integration has no configuration parameters
```

Reply:
The integration has configuration parameters. Documentation is here:
https://github.com/home-assistant/home-assistant.io/blob/82e9afb0f9600ae7cb05b0f2c282bdcec87f3c0f/source/_integrations/lcn.markdown?plain=1#L44

Reply:
The link you mention is covered by `docs-installation-parameters`. The `docs-configuration-parameters` is for the options_flow, which this does not have.